telesto/models/classification.py

```diff
@@ -29,7 +28,7 @@ def _load_model(self, model_path: str):
     def predict(self, input_list: List[np.ndarray]) -> np.ndarray:
         raise NotImplemented
 
-    def _preprocess(self, doc):
+    def _preprocess(self, doc: dict) -> List[np.ndarray]:
```

Comment:
I expect these to be the same in `SegmentationModelBase`, but not sure. Maybe this part of the code could be reused.